.github/workflows/license_check.yml

```diff
@@ -0,0 +1,56 @@
+name: Check licenses
+
+on: 
+  workflow_dispatch:
+  push:
+    paths-ignore:
+      - README.md
```

Comment:
issue: This should be removed because this is a reusable workflow now

Reply:
removed